src/sentry/api/endpoints/event_ai_suggest.py

```diff
@@ -0,0 +1,217 @@
+import logging
+import random
+from collections import OrderedDict
+
+import openai
+from django.conf import settings
+from django.http import HttpResponse
+
+from sentry import eventstore, features
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases.project import ProjectEndpoint
+from sentry.api.exceptions import ResourceDoesNotExist
+from sentry.types.ratelimit import RateLimit, RateLimitCategory
+from sentry.utils import json
+from sentry.utils.cache import cache
+
+logger = logging.getLogger(__name__)
+
+
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+# XXX: this is pretty ugly
+openai.api_key = settings.OPENAI_API_KEY
+
+FUN_PROMPT_CHOICES = [
+    "[haiku about the error]",
+    "[hip hop rhyme about the error]",
+    "[4 line rhyme about the error]",
+    "[2 stanza rhyme about the error]",
+    "[anti joke about the error]",
+]
+
+PROMPT = """\
+You are an assistent that analyses software errors, describing the problem with the follwing rules:
```

Comment:
Nit: assistant, following

Not sure if typos affect the prompt but probably worth fixing.